packages/calcite-components/src/components/tooltip/tooltip.tsx

```diff
@@ -56,8 +56,12 @@ export class Tooltip implements FloatingUIComponent, OpenCloseComponent {
   /** Closes the component when the `referenceElement` is clicked. */
   @Prop({ reflect: true }) closeOnClick = false;
 
-  /** Accessible name for the component. */
-  @Prop() label!: string;
+  /**
+   * Accessible name for the component.
+   *
+   * @deprecated No longer necessary.
```

Comment:
Can we add some info on this no longer being needed because the label will come from the content/setting ARIA props on it not being recommended?

Reply:
From the changelog perspective, could you also revisit the PR title to include something about this?

Reply:
WDYT about:

```
@deprecated The component's content provides the description, so this should no longer necessary.
```

?

Deferring to @geospatialem for some docspertise™.

Reply:
How about..

> @deprecated This is no longer necessary because the component's content provides the description.

I like "is" better than "should" and this ordering. Should sounds less confident :D

Reply:
WDYT about some context as to what the `label` would provide if used? Something like:

> @deprecated No longer necessary. Overrides the context of the component's description, which could confuse assistive technology users.

Reply:
sounds good to me